ts/src/renderer/phaser/classes/PhaserEntity.ts

```diff
@@ -32,38 +23,26 @@ abstract class PhaserEntity extends Phaser.GameObjects.Container {
 		this.scaleListener = entity.on('scale', (data) => {
```

Comment:
I think @pavle-goloskokovic suggested these be:
```js
this.scaleListener = entity.on('scale', this.scaleEntity);
```

Reply:
currently can't call .on on an instance of IgeEntityPhysics with 3 args in this manner. I posted more in our chat but see the .on method of IgeEntityPhysics for the reason.

Reply:
took a look at this, box2d expects second argument target to be a `string` while eventing class expects callback method
so an elegant fix would be to check if second argument is of type string in which case do the box2d logic, otherwise do the regular event handling